ushahidi/platform-client
# Displaying Tweets from Twitter

This pull request makes the following changes:
- Show embedded Tweets directly from Twitter in data-view:
 - In post-list
 - In post-detailed-view
 - When editing a post
- Adds a new version of PL to adjust margins for the embedded Tweets 


Testing checklist:
- Make sure the deployment has posts with Twitter as datasource
- Go to data-view
[ ] The tweets are embedded directly from Twitter in the post-list
- Click on a post with Twitter as datasource
[ ] The tweet-content is embedded in the detailed view to the right
- Click on "edit"
[ ]  The Tweet  is still visible above the "Structure this post" info
- Select a survey for the post
- [ ] The Tweet is still visible at the top of the form
- Look in the end of the detailed view
[ ] No "conversation with author" is visible in edit-view
[ ] No "conversation with author" is visible in detailed-view

Fixes ushahidi/platform#3334 .

Ping @ushahidi/platform
